fabtests/multinode/include/core.h

```diff
@@ -44,21 +44,35 @@
 
 #define PM_DEFAULT_OOB_PORT (8228)
 
+enum multi_xfer{
+	msg = 0,
+	rma = 1,
```

Comment:
no need to set explicitly.  Please add some other prefix here, so that 'msg' and 'rma' don't show up in the code without any other meaning.